code/game/gamemodes/objective_items.dm

```diff
@@ -3,10 +3,12 @@
 /datum/objective_item
 	var/name = "A silly bike horn! Honk!"
```

Comment:
I'm going to sound like a little shit here but we could document these since it may be the last time for another thousand years that we look at any of the variable documentation.